game/stringtables/entities_de.str

```diff
@@ -1,214 +1,235 @@
-Tags (do not translate)                                                    DE
 // Game Mechanics
-AttackType_none                                                            Keine
-AttackType_melee                                                           Nahkampf
-AttackType_ranged                                                          Aus der Ferne
-
-EffectType_Attack                                                          Angriff
-EffectType_AttackDamage                                                    Angriffsschaden
-EffectType_Magic                                                           Magie
-EffectType_StatusBuff                                                      Buff
-EffectType_StatusDebuff                                                    Debuff
+AttackType_none														Keine
+AttackType_melee													Nahkampf
+AttackType_ranged													Aus der Ferne
+
+EffectType_Attack													Angriff
+EffectType_AttackDamage												Angriffsschaden
+EffectType_Magic													Magie
+EffectType_StatusBuff												Buff
+EffectType_StatusDebuff												Debuff
 EffectType_StatusDisable
 EffectType_StatusStealth
-EffectType_Push                                                            Offensive
-EffectType_Splash                                                          Flächenschaden
-EffectType_DOT                                                             Zeitschaden
-EffectType_Buff                                                            Buffs
+EffectType_Push														Offensive
+EffectType_Splash													Flächenschaden
+EffectType_DOT
+EffectType_Buff
```

Comment:
Why was the value removed here? Am i missing something? Same for _DOT

Reply:
I've used a self written python script to merge the english and german file. 
I didn't account for the fact, that the german file might have a value, that the english one has not set at all.
The translation for those two are correct, as they were originally.